src/bucket.ts

```diff
@@ -2752,9 +2752,10 @@ class Bucket extends ServiceObject {
    *
    * @see [Storage Classes]{@link https://cloud.google.com/storage/docs/storage-classes}
    *
-   * @param {string} storageClass The new storage class. (`multi_regional`,
-   *     `regional`, `standard`, `nearline`, `coldline`, or
-   *     `durable_reduced_availability`)
+   * @param {string} storageClass The new storage class. (`standard`,
+   *     `nearline`, `coldline`, or `durable_reduced_availability`).
+   *     **Note:** The legacy storage classes `multi_regional` and `regional`
```

Comment:
@JesseLovelace I thought about remove the available storage classes in favor of just linking to the upstream docs. I could see the list going stale and getting overlooked. WDYT?

Reply:
That's a valid concern, I'm good with either approach